controllers/controllers.js

```diff
@@ -14,4 +14,20 @@ function sendAllEndpoints(req, res, next){
     res.status(200).send(endpoints)
 }
 
-module.exports = { sendTopics, sendAllEndpoints }
\ No newline at end of file
+function sendArticleById(req, res, next){
+    const {article_id} = req.params;
+
+    return getArticlebyId(article_id)
+    .then((article) => {
+        res.status(200).send({article})
+    })
+    .catch((err) => {
```

Comment:
This error handling would be nicer if it was included in our error handling middleware chain